### DISCUSSION TITLE
Fontsource API

### DISCUSSION THREAD

**Discussion by nathalie-lammelin** [fontsource/fontsource#510] [Q&A] [answered]
I use the API (https://api.fontsource.org/v1/fonts) but since last friday the server does not answer, it seems to be down.
Is there another way to get the json returned from the API ?

**Comment by jwr1** [comment, marked as answer]
The API is back online! Thank you for notifying us of this issue. The API is not supposed to have breaking changes, the version route number would be incremented for that, so if you notice this again, please let up know.

**Reply by ayuhito** [reply]
I think it'd be nice for us to get some sort of status monitor for the API so we are notified sooner when something like this happens ^^

**Reply by nathalie-lammelin** [reply]
Thanks a lot. I was able to access the API this morning :)
